insights/parsers/up2date_log.py

```diff
@@ -0,0 +1,7 @@
+from .. import LogFileOutput, parser
+
+
+@parser('up2date.log')
+class Up2dateLog(LogFileOutput):
+    """Class for parsing the log file: ``/var/log/up2date`` """
+    pass
```

Comment:
The new sub-classes inherits from LogFileOutput, you should check if the default `time_format` can cover the actual time format of the new log-file. The default time format is 

`time_format = '%Y-%m-%d %H:%M:%S'`

It does not match with the time stamp string in up2date. Please modify it correspondingly and use `get_after` in test to verify it.

Reply:
I will modify this

Reply:
done